### ISSUE TITLE
Fixed "disable dynamic bones mod" button position

### ISSUE THREAD

**Issue by EaranMaleasi** [charlesdeepk/MultiplayerDynamicBonesMod#1]
Tried out the mod, and it works fine so far. Thank you for that. 

Just a little thing that I noticed, because I already used emmVRC is, that the button position overlaps with the emmVRC default. This is not a big issue, because emmVRC allows to change the position, but I think this would also be a great addition for your mod, so one can change the position if there is another mod not able to do so. 

So in short, what I propose is:
Add the possibility to define the button position in the .ini file.

**Comment by charlesdeepk** [comment]
I have to check the UI system but I'll add the option to move the button on a later release. Thanks for noticing it.

**Comment by charlesdeepk** [comment]
Proposal included in commit bd992ad538f7e5fdd3939ae8d18bf19a82fba0a0. Closing the issue.
